### DISCUSSION TITLE
ARMv6 support

### DISCUSSION THREAD

**Discussion by caliangxie** [nodejs/undici#1595] [Q&A]
Is it possible for undici to work on ARMv6? I am trying to run a Discord bot on a raspberry pi 2 using Discord.js. One of Discord.js dependencies is undici which requires wasm that I cannot run on my raspberry pi. It seems like there is no fallback for llhttp however.
Are there any alternatives to making this work for me?

**Comment by mcollina** [comment]
Unfortunately no, there aren't any alternatives without wasm. Why isn't wasm available in your Node.js build?

**Reply by caliangxie** [reply]
The raspberry pi is running on armv6 which doesn't seem to have active support for wasm any more.

**Reply by mcollina** [reply]
what version of Node.js are you running?

**Reply by caliangxie** [reply]
16.9.1

**Reply by mcollina** [reply]
@nodejs/platform-arm could you confirm that we are building without wasm for ARMv6 / RaspberryPI?

**Reply by rvagg** [reply]
Depends on where you're getting the build from, unofficial-builds doesn't seem to be explicitly disabling it: https://github.com/nodejs/unofficial-builds/blob/main/recipes/armv6l/run.sh, perhaps this is implicit via v8 somehow?

**Comment by sxa** [comment]
The raspberry pi2 should support ARMv7 without problems - why do you need armv6 support for that board?
